### PR TITLE
Added DL cases for python cdk

### PR DESCRIPTION
**Pull request by shreyabhagat1**
Added non compliant and compliant samples for python cdk detectors.

Note: All the test cases  have 100 % recall and precision.

### CODE REVIEW

**Review by saurabh-amazon — APPROVED** [review]
Reviewed all changes. Changes look fine. Approved.

File: cdk_python/src/detectors/python-cdk-document-db-cluster-encryption-at-rest/python-cdk-document-db-cluster-encryption-at-rest_non-compliant.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+from aws_cdk.aws_ec2 import InstanceType,InstanceClass,InstanceSize,Vpc
+from aws_cdk.aws_docdb import DatabaseCluster, core
+
+class CdkStarterStack(cdk.Stack):
+    def __init__(self, scope: cdk.App, id: str):
+      super(scope, id)
+
+      vpc = Vpc(self, "MyVpc")
+
+	    # Noncompliant: Disables storage encryption, potentially exposing sensitive data.
```
Comment (grl-wang) [inline cdk_python/src/detectors/python-cdk-document-db-cluster-encryption-at-rest/python-cdk-document-db-cluster-encryption-at-rest_non-compliant.py:15]: pls fix the indentation, also check other places.

Comment (shreyabhagat1) [reply cdk_python/src/detectors/python-cdk-document-db-cluster-encryption-at-rest/python-cdk-document-db-cluster-encryption-at-rest_non-compliant.py:15]: Sure!

**Review by saurabh-amazon — APPROVED** [review]
Reviewed corrections. Necessary corrections have been made. Approved.

**Review by grl-wang — APPROVED** [review]

**Pull request merged**